pom.xml

```diff
@@ -390,7 +390,7 @@
         <configuration>
           <configLocation>${checkstyle.path}tachyon_checks.xml</configLocation>
           <includeTestSourceDirectory>true</includeTestSourceDirectory>
-          <excludes>**/tachyon/thrift/**</excludes>
+          <excludes>**/org/apache/jsp/**,**/tachyon/thrift/**</excludes>
```

Comment:
Yes, it is the way: http://stackoverflow.com/questions/2362652/excluding-classes-in-maven-checkstyle-plugin-reports